torch/ao/quantization/fx/_lower_to_native_backend.py

```diff
@@ -130,9 +137,149 @@ def _lower_weighted_ref_module(model: QuantizedGraphModule) -> QuantizedGraphMod
         model.recompile()
     return model
 
+def _lower_quantized_binary_op(
+    model: QuantizedGraphModule,
+    qconfig_map: Dict[str, QConfigAny]
+) -> QuantizedGraphModule:
+    modules = dict(model.named_modules(remove_duplicate=False))
+
+    def get_bop_patterns(bop: Callable) -> Pattern:
+        return [
+            (torch.quantize_per_tensor,
+             (bop, "dequantize", "dequantize"),
+             MatchAllNode, MatchAllNode, MatchAllNode)
+            for bop_pattern in [
+                (bop, "dequantize", "dequantize"),
+                (bop, MatchAllNode, "dequantize"),
+                (bop, "dequantize", MatchAllNode)
+            ]
+        ]
+
+    def get_bop_relu_patterns(bop: Callable) -> List[Pattern]:
```

Comment:
nit: I wonder if there's a way to merge these two and still be clear, maybe like:
```
def get_bop_patterns(bop):
    patterns = []
    for bop_pattern in [...]:
        patterns.append(
            (torch.quantize_per_tensor,
                bop_pattern,
                MatchAllNode, MatchAllNode, MatchAllNode))
        for relu_op in [...]:
            patterns.append(
                (torch.quantize_per_tensor,
                     (relu_op, bop_pattern),
                     MatchAllNode, MatchAllNode, MatchAllNode))
    return patterns
```
What do you think?

Reply:
makes sense